### DISCUSSION TITLE
Latex counter between chapters

### DISCUSSION THREAD

**Discussion by MichelDSchmid** [quarto-dev/quarto-cli#4799] [Q&A] [answered]
Hi all

I'm wondering if it's possible to create in Quarto fro PDF-Output (Latex) something similar like:

`
\include{chapter/ladungSpannungStromDichte}
\stepcounter{chapshift}

\include{chapter/ohmKirchhoff}
\stepcounter{chapshift}
\include{chapter/resistor}
`
I'm doing this to increment an counter which changes the color of each new chapter. This must happen between the chapters, otherwise the colors aren't changed correctly.

Any hints?

kind regards
Mike

**Comment by dragonstyle** [comment, marked as answer]
One approach I think of is the use an include at the start of each chapter, like:

```markdown
{{< include colorize-chapter.qmd >}}
# Introduction

This is a book created from markdown and executable code.

See @knuth84 for additional discussion of literate programming.
```

which points to a file containing a raw latex block, like:

````markdown
```{=latex}
\stepcounter{chapshift}
```
````

Would that work?

**Reply by MichelDSchmid** [reply]
Thanks for your answer, I already tried this without any success.

But I could figure out the solution 10 minutes ago.... 
I had to change the \include{}..... which generates new pages at the end of the command to a 
\input{}
\newpage
\stepcounter{chapshift}

\include and \input are almost the same,.... but only almost.

Regards
Mike
